src/index.html

```diff
@@ -198,7 +198,7 @@ <h2>Made with love.</h2>
           inspiring me to do this.</small></p>
     </div>
   </section>
-  <template>The boogeyman wrote all changes to the **master server** as he was all out of fucks to give. Thus, the slaves were read-only copies of master.</template>
+  <template>The boogeyman wrote all changes to the **master server**, as he was tired of waiting for those buttheads in IT. Thus, the slaves were read-only copies of master.</template>
```

Comment:
Alright, so here's what's up --
A few of the lines I tried didn't render quite perfectly, so I cycled through util I both had the visually correct highlighting (on my computer /w chrome), and used the word `butt` (here, `buttheads`).

It's a bit awkwardly phrased, but I hope it gets the message across in a _"good enough for government work"_ kind of way.

What do you think?

Reply:
Nice! I think the text being nonsensical is good!